### ISSUE TITLE
Lesson 7, Practice 2 — Turtle animation not working as expected. 

### ISSUE THREAD

**Issue by miwidev** [GDQuest/learn-gdscript#597]
**Bug description**
When using the jump() function instead of the intended "position.x"-solution, the output animation behaves in a weird manner.

**To Reproduce**
Steps to reproduce the bug:
1. Go to Lesson 7, Practice 2.
2. Enter the code into the editor.
3. Replace Line 6 and Line 9 with "jump(200, 0)"

**Expected behavior**
I expected the turtle to draw a square, jump to the right, and then draw the next square, and then jump again, and then draw the last square.

**Screenshots**
![bild](https://user-images.githubusercontent.com/60471703/180116147-fcb97f4f-4f57-40a6-9e60-a5fb816afbcf.png)

**Information about your device (please complete the following information):**
 - Operating System: [Windows 10]
 - Browser: [Firefox]

**Additional context**
The lesson is all about assigning values onto variables. Because of this, the animation for this lesson wasn't made for this solution in mind and because of this bugs out.

**Comment by 595489** [comment]
![image](https://user-images.githubusercontent.com/69585797/185818032-0434aa3e-2e94-4271-b293-77e2c4d3aa50.png)

There is a similar issue when making the code for the square yourself

**Comment by NathanLovato** [comment]
Yes, you need to follow the practice's requirements right now. We're trying to have you practice specific techniques you just learned and test specifically for that. This applies to almost every practice in the app.

**Comment by NathanLovato** [comment]
I just worked a bit on this and ran the animations through the debugger, but I couldn't find the cause just yet so I'm moving this to a future as this is a minor issue.

**Comment by danijmoss** [comment]
[2022-12-25-20-33.log](https://github.com/GDQuest/learn-gdscript/files/10301004/2022-12-25-20-33.log)

Was working on this today and had this code end up coming through as correct. I'm not sure if it should be labeled as correct though. 

Here's my code in text form in case the image doesn't come through:
func run():
	position.x = 100
	position.y = 100
	draw_rectangle(100, 100)
	
	position.x = 200
	draw_ractangle(100, 100)
	
	position.x = 300
	draw_rectangle(100, 100)
	
	position.x = 500
	draw_rectangle(100, 100)
